src/ContainerAppHelper.ts

```diff
@@ -30,11 +30,10 @@ export class ContainerAppHelper {
         containerAppName: string,
         resourceGroup: string,
         environment: string,
-        imageToDeploy: string,
         optionalCmdArgs: string[]) {
         toolHelper.writeDebug(`Attempting to create Container App with name "${containerAppName}" in resource group "${resourceGroup}" based from image "${imageToDeploy}"`);
```

Comment:
`imageToDeploy` is still referenced in this debug message even though we removed the argument

Reply:
Removed `imageToDeploy` from the debug message.